src/nodedef.cpp

```diff
@@ -339,15 +344,22 @@ void ContentFeatures::deSerialize(std::istream &is)
 		groups[name] = value;
 	}
 	drawtype = (enum NodeDrawType)readU8(is);
+
+	bool ignore_culling = ((version <= 26) &&
+			((drawtype == NDT_MESH) ||
```

Comment:
No. None of these drawtypes are adjacent in the enum NodeDrawType. (they're 2, 9, 14 and 16)